src/components/dev-hub/twitch-fallback-card.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Card from './card';
+import { screenSize } from './theme';
+
+const StyledIframe = styled('iframe')`
+    height: 100%;
+    width: 100%;
+    border: none;
+`;
+
+const TwitchCard = styled(Card)`
+    height: 100%;
+    width: 100%;
+    @media ${screenSize.upToLarge} {
+        height: 300px;
```

Comment:
Was getting some percentage responsiveness issues since this isnt an `img` but didn't think it was necessary to dig into the media block and card for this one case